.github/auto-merge.yml

```diff
@@ -10,10 +10,6 @@
     dependency_name: Fody
     update_type: all
 
-- match:
-    dependency_name: GitVersionTask
-    update_type: all
-
```

Comment:
We only have two references:

- https://github.com/Particular/Particular.Packaging/blob/ab4cb75b9fdb6f8d63805a2828fca56996f10c7e/src/Particular.Packaging/Particular.Packaging.csproj#L17
- another in a private repo

The risk of a bad GitVersion making it into all our repos via `Particular.Packaging` outweighs the benefit of merging two PR's automatically. Note that our CI build proves nothing. A new version of GitVersion may just produce garbage versions.

Reply:
Can you clarify? What does it mean to produce garage versions? Is this package automatically build after every PR? Otherwise a TF working on it should spot that something is off while releasing.

Reply:
What I mean is: a new version of `GitVersionTask` may be broken in some way, and our CI will not detect it. The new version of `GitVersionTask` should be verified by someone reviewing the PR before merging it. Leaving this detection until the next time someone want to release doesn't seem wise. By that time the new version of `GitVersionTask` would have made it into all our repos via `Particular.Packaging` and would have to be reverted in all of them.

But in any case, I think this discussion is moot: we only reference `GitVersionTask` in a single repo (the reference in the private repo can probably be removed). So is it worth taking the risk of automerging an update to `GitVersionTask` to save one manual merge?

Reply:
> Leaving this detection until the next time someone want to release doesn't seem wise

I would argue the contrary. If a release of GitVersionTask is flawed but does not show as flawed in our CI, it is far better to have a taskforce looking at what could have gone wrong than someone who has some time to look at something on the small tasks board. The assumption you're making is that someone looking at the release is able to spot every mistake or problem.

The contrary would be a taskforce working on updating the packaging of our tools and they would spend presumably be spending more than a max of 8 hours on it. The taskforce would have the mindset, the context, and - more importantly - be working towards an actual end goal that they would presumably be testing more thoroughly than someone taking a small task.

Does it not make more sense to have more eyes looking at something that isn't working than a single person doing some extra work in a context-switched mode?

Reply:
reverting after discussion